core/src/main/java/it/pagopa/selfcare/mscore/core/strategy/factory/OnboardingInstitutionStrategyFactory.java

```diff
@@ -138,10 +138,11 @@ private Consumer<OnboardingInstitutionStrategyInput> verifyManagerAndPersistWith
 
     private Consumer<OnboardingInstitutionStrategyInput> verifyManagerAndDelegateAndPersistWithDigest() {
         return strategyInput -> {
-            if(!InstitutionType.SA.equals(strategyInput.getOnboardingRequest().getInstitutionUpdate().getInstitutionType())) {
-                OnboardingInstitutionUtils.validatePaOnboarding(strategyInput.getOnboardingRequest().getBillingRequest());
+            if(strategyInput.getOnboardingRequest().getInstitutionUpdate().getInstitutionType().equals(InstitutionType.SA)
+            || strategyInput.getOnboardingRequest().getInstitutionUpdate().getInstitutionType().equals(InstitutionType.PT)){
+                OnboardingInstitutionUtils.validateVatNumber(strategyInput.getOnboardingRequest().getBillingRequest().getVatNumber());
```

Comment:
it might be occurred an nullpointerexception...you could use only one method and pass a boolean if check also the getRecipientCode. For example:

```
 public static void validateBilling(Billing billing, boolean checkRecipientCode) {
        if (billing == null
                || StringUtils.isEmpty(billing.getVatNumber())
                || (checkRecipientCode && StringUtils.isEmpty(billing.getRecipientCode()))) {
.....
```